web/src/styles/scss/secondary-nav.scss

```diff
@@ -1,3 +1,7 @@
+.button-description {
```

Comment:
Instead of having this extra custom class you could use the design systems utility class: `ds-u-margin-bottom--2`

https://design.cms.gov/utilities/margin/?theme=core#margin-bottom

Reply:
Great suggestion, thanks! I updated the code!